src/qt/bitcoin.cpp

```diff
@@ -302,7 +302,7 @@ void BitcoinCore::shutdown()
         threadGroup.join_all();
         Shutdown();
         qDebug() << __func__ << ": Shutdown finished";
-        Q_EMIT shutdownResult(1);
```

Comment:
Was a bug, or is a bug? Because it is still not called with (0), and I am not sure under what circumstances it should be, seeing that the runaway exception signal is called when shutdown fails.